config-linux.md

```diff
@@ -494,6 +494,98 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.
+
+The OCI runtime MUST ensure that the needed cgroup controllers are enabled for the cgroup.
+
+Configuration unknown to the runtime MUST still be written to the relevant file.
+
+The runtime MUST generate an error when the configuration refers to a cgroup controller that is not present or that cannot be enabled.
+
+### Example
+
+```json
+"unified": {
+    "io.max": "259:0 rbps=2097152 wiops=120\n253:0 rbps=2097152 wiops=120",
+    "hugetlb.1GB.max": "1073741824"
+}
+```
+
+If a controller is enabled on the cgroup v2 hierarchy but the configuration is provided for the cgroup v1 equivalent controller, the runtime MAY attempt a conversion.
+
+If the conversion is not possible the runtime MUST generate an error.
+
+### Example
+
+The provided configuration is valid for cgroup v1, where the CPU controller `shares` value has the range `[2-262144]`:
+
+```json
+"cpu": {
+    "shares": 1024
+}
+```
+
+If the CPU controller is enabled on the cgroup v2 hierarchy, the value is converted to the equivalent `cpu.weight` that has a range of `[1-10000]`:
+
+```json
+"unified": {
+    "cpu.weight": "1"
+}
+```
+
+When converting from cgroup v1 to cgroup v2, the runtime SHOULD convert in the following way:
+
+## Memory controller
+
+| OCI (x) | cgroup 2 value (y) | conversion  |   comment |
```

Comment:
Shouldn't this be `cgroup v1 value (x)` instead of `OCI (x)`? (also in the tables below)

Reply:
yes, that is better.  Fixed

Reply:
Correspondingly, `cgroup 2 value (y)` --> `cgroup v2 value (y)` would make a lot of sense.

Reply:
thanks, fixed.

Reply:
I am still not fully convinced the OCI runtime specs should mandate how the conversion is performed.  There are other valid ways to convert from one range to the other.

I'd prefer if it is just a suggestion.  @AkihiroSuda @cyphar what do you think?

Reply:
Personally, I think that linear transform of the respective ranges is the only way that satisfies the principle of least surprise, and would therefore be fine with mandating that.

Reply:
the linear transform doesn't keep the default value unchanged though.

`cpu.shares=1024` will be converted to 39 (or 40), instead of the default `100` for `cpu.weight`.

So there are better ways of converting it, if keeping the default value is important.

I still think the OCI runtime specs should not mandate how the conversion is performed, it looks more like an implementation detail left to the OCI runtime.